extension.meta.xml

```diff
@@ -14,6 +14,9 @@
 		</author>
 	</authors>
 	<releases>
+		<release version="1.7.2" date="2017-05-19" min="2.5.x" max="4.x.x" php-min="5.6.x" php-max="7.x.x">
```

Comment:
@michael-e [we agreed on this](https://github.com/symphonists/symphony-extensions-network/issues/17) yes, but it was never implemented in an "official" way ;) I am still all for it !